src/app/metadata/features/metadata-browser/metadata-browser.component.html

```diff
@@ -1,28 +1,66 @@
 <div class="grid grid-cols-12 grid-rows-none gap-5">
   <div class="col-span-9 col-start-4">
+    @if (lastSearchQuery() !== '') {
```

Comment:
```suggestion
    <mat-chip class="float-right m-1"
      >Total Datasets: {{ searchResults().count }}</mat-chip
    >
    <mat-chip-set aria-label="search and filters">
    @if (lastSearchQuery() !== '') {
```

Reply:
Looks better to use a mat-chip-set when it comes to separation. I also fixed up the floating of the total datasets chip, and added a margin to make it equivalent to the other chips.